third_party/ibis/ibis_addon/operations.py

```diff
@@ -188,13 +196,23 @@ def sa_format_hashbytes_postgres(translator, expr):
     hash_func = sa.func.sha256(convert)
     return sa.func.encode(hash_func, sa.sql.literal_column("'hex'"))
 
+def sa_format_to_char_oracle(translator, expr):
```

Comment:
We can just make this one function `sa_format_to_char()` and set the OracleExprTranslator and PostgreSQLExprTranslator  registry to the same function.

Reply:
Done